modules/salt/pages/formula-yomi.adoc

```diff
@@ -0,0 +1,153 @@
+[[yomi-formula]]
+= Yomi Formula
+
+Yhe Yomi (yet one more installer) installer for {suse} and openSUSE operating systems is configured using formulas with forms.
+
+The ``yomi-formula`` package provides these formulas:
+
+* Yomi
+* Yomi Storage
+* Yomi Bootloader
+* Yomi Software
+* Yomi Services
+* Yomi Users
+
+
+.Procedure: Install the Yomi Formulas with Forms
+. On the {productname} Server, at the command prompt, as root, install the ``yomi-formula`` package:
++
+----
+zypper in yomi-formula
+----
+. In the {susemgr} {webui}, open the details page for the Salt master, and navigate to the Formulas tab.
+. Check the [guimenu]``Installer`` box, and click btn:[Save].
+. Apply the highstate.
+
+
+
+.Procedure: Configuring the Yomi Formula
+. Navigate to the menu:Formulas[Yomi] tab, and set these parameters in the [guimenu]``General Configuration`` section:
+* Check the [guimenu]``Events`` box to allow monitoring.
+* In the [guimenu]``Reboot`` field, select [systemitem]``yes`` to instruct the client to reboot after installation.
+* Check the [guimenu]``Snapper`` box if you are using the btrfs file system on the client.
+* In the [guimenu]``Locale`` field, select the region and encoding for systemd to use on the client (for example: [systemitem]``en_US.utf8`` for American English and UTF-8).
```

Comment:
Better call it "English (US)" (see Gnome settings).  en_CA is also American English.